supervisor/container.go

```diff
@@ -8,8 +8,11 @@ import (
 	"os"
 	"os/exec"
 	"path/filepath"
+	"strings"
 	"time"
 
+	"github.com/docker/docker/pkg/mount"
+	"github.com/docker/docker/pkg/symlink"
```

Comment:
please check the travis error

supervisor/container.go:14:2: cannot find package "github.com/docker/docker/pkg/mount"